lib/scan_beacon/eddystone_url_beacon.rb

```diff
@@ -0,0 +1,57 @@
+module ScanBeacon
+  # Convenience class for constructing & advertising Eddystone-URL frames
+  class EddystoneUrlBeacon < Beacon
+
+    attr_accessor :url
```

Comment:
Yes, I think you're right!  I'm working on refining this PR further, so I'll take this into account.